src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -855,37 +857,44 @@ public void onChunkSatelliteWindowOpened(ChunkSatelliteWindowOpenedEvent event)
    @Override
    public void onRenderFinished(RenderFinishedEvent event)
    {
-      // single shot rendering of output line widgets (we wait until after the
-      // first render to ensure that ace places the line widgets correctly)
-      if (initialChunkDefs_ != null)
-      {
-         for (int i = 0; i < initialChunkDefs_.length(); i++)
-         {
-            createChunkOutput(initialChunkDefs_.get(i));
-         }
-         // if we got chunk content, load initial chunk output from server
-         if (initialChunkDefs_.length() > 0)
-            loadInitialChunkOutput();
-
-         initialChunkDefs_ = null;
-         
-         // sync to editor style changes
-         editingTarget_.addEditorThemeStyleChangedHandler(
-                                       TextEditingTargetNotebook.this);
-         
-         // read and/or set initial render width
-         lastPlotWidth_ = notebookDoc_.getChunkRenderedWidth();
-         if (lastPlotWidth_ == 0)
+      dependencyManager_.withRMarkdown("R Notebook",
+         "Opening R Notebook", new Command() {
+         @Override
+         public void execute()
```

Comment:
Use the `CommandWithArg<Boolean>` variant here so that the output repopulation happens even if the package versions aren't sufficient.